sources/packages/backend/apps/workers/src/controllers/disbursement/_tests_/e2e/disbursement.controller.saveDisbursementSchedules.e2e-spec.ts

```diff
@@ -344,6 +344,119 @@ describe("DisbursementController(e2e)-saveDisbursementSchedules", () => {
     assertOveraward(overawards, "CSLP", 300);
   });
 
+  it("Should assert disbursements do not have estimated awards and disbursement value amounts are zero.", async () => {
+    // Arrange
+    const savedUser = await db.user.save(createFakeUser());
+    const savedStudent = await db.student.save(createFakeStudent(savedUser));
+    const fakeApplication = createFakeApplication(
+      { student: savedStudent },
+      { initialValue: { applicationStatus: ApplicationStatus.Completed } },
+    );
+    fakeApplication.applicationNumber = "OA_TEST003";
+    const savedApplication = await db.application.save(fakeApplication);
+    // Original assessment.
+    const fakeOriginalAssessment = createFakeStudentAssessment({
+      auditUser: savedUser,
+    });
+    fakeOriginalAssessment.application = savedApplication;
+    // Original assessment - first disbursement (Pending).
+    const firstSchedule = createFakeDisbursementSchedule(
+      {
+        disbursementValues: [
+          createFakeDisbursementValue(
+            DisbursementValueType.CanadaLoan,
+            "CSLF",
+            0,
+          ),
+          createFakeDisbursementValue(DisbursementValueType.BCLoan, "BCSL", 0),
+        ],
+      },
+      { initialValues: { hasEstimatedAwards: false } },
+    );
+    firstSchedule.disbursementScheduleStatus =
+      DisbursementScheduleStatus.Pending;
+    // Original assessment - second disbursement (Pending).
+    const secondSchedule = createFakeDisbursementSchedule(
+      {
+        disbursementValues: [
+          createFakeDisbursementValue(
+            DisbursementValueType.CanadaGrant,
+            "CSLF",
+            0,
+          ),
+          createFakeDisbursementValue(DisbursementValueType.BCLoan, "BCSL", 0),
+        ],
+      },
+      { initialValues: { hasEstimatedAwards: false } },
+    );
```

Comment:
First and second disbursement schedules are created by the worker. So we must not create the disbursements here.